.env.example

```
@@ -93,6 +93,8 @@ OPENAI_TEXT_MAX_RETRIES=1
 
 # Optional: bypass daily Messenger quota for these IDs (comma-separated PSIDs or hashed userKeys).
 MESSENGER_QUOTA_BYPASS_IDS=
+# Optional: ask Messenger users for explicit 30-day photo reuse consent.
+ENABLE_FACE_MEMORY=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENABLE_FACE_MEMORY key should go before the MESSENGER_QUOTA_BYPASS_IDS key

(UnorderedKey)